mpa/seg/stage.py

```diff
@@ -197,9 +197,10 @@ def configure_task_classes(self, cfg, task_adapt_op):
 
         # Model classes
         if task_adapt_op == 'REPLACE':
-            if len(data_classes) == 0:
-                raise ValueError('Data classes should contain at least one class!')
-            model_classes = data_classes.copy()
+            if len(data_classes) == 1:
+                model_classes = org_model_classes.copy()
+            else:
+                model_classes = data_classes.copy()
```

Comment:
What do you mean by "== 1"?

Reply:
always adding bg. 1 is bg size.

Reply:
Thanks. Please add some comments for understanding.